frontend/src/app/modules/global_search/global-search-work-packages.component.ts

```diff
@@ -26,46 +26,33 @@
 // See docs/COPYRIGHT.rdoc for more details.
 // ++
 
-import {
-  AfterViewInit,
-  ChangeDetectorRef,
-  Component,
-  ElementRef,
-  OnDestroy,
-  OnInit,
-  Query,
-  Renderer2
-} from '@angular/core';
+import {AfterViewInit, ChangeDetectorRef, Component, ElementRef, OnDestroy, OnInit, Renderer2} from '@angular/core';
 import {FocusHelperService} from 'app/modules/common/focus/focus-helper';
 import {I18nService} from 'app/modules/common/i18n/i18n.service';
-import {DynamicBootstrapper} from "app/globals/dynamic-bootstrapper";
 import {HalResourceService} from "app/modules/hal/services/hal-resource.service";
 import {GlobalSearchService} from "core-app/modules/global_search/services/global-search.service";
-import {untilComponentDestroyed} from "ng2-rx-componentdestroyed";
-import {QueryResource} from "app/modules/hal/resources/query-resource";
 import {WorkPackageFiltersService} from "app/components/filters/wp-filters/wp-filters.service";
 import {UrlParamsHelperService} from "app/components/wp-query/url-params-helper";
 import {WorkPackageTableConfigurationObject} from "core-components/wp-table/wp-table-configuration";
-import {cloneHalResource} from "core-app/modules/hal/helpers/hal-resource-builder";
 import {IsolatedQuerySpace} from "core-app/modules/work_packages/query-space/isolated-query-space";
-import {QueryFilterInstanceResource} from "core-app/modules/hal/resources/query-filter-instance-resource";
 import {WorkPackageViewFiltersService} from "core-app/modules/work_packages/routing/wp-view-base/view-services/wp-view-filters.service";
 import {debounceTime, distinctUntilChanged, skip} from "rxjs/operators";
 import {combineLatest} from "rxjs";
+import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
 
 export const globalSearchWorkPackagesSelector = 'global-search-work-packages';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/85e9c35a9c9361e0038637188a645cba25065d78/frontend/src/app/modules/global_search/global-search-work-packages.component.ts#L44"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>